docs/developer/centrifuge-api/overview/index.md

```diff
@@ -0,0 +1,203 @@
+---
+id: overview
+title: Overview
+category: subpage
+contributors: <Filippo:filippo@embriotech.ch>
+---
+
+# Overview
+
+Centrifuge API leverages [GraphQl](https://graphql.org/). GraphQL is an open-source query language standard for APIs and a runtime. It uses a strongly-typed approach to querying large and complex sets of data. It provides a more efficient, powerful, and flexible alternative to the traditional REST API approach. With GraphQL, clients can request exactly the data they need, and nothing more, which reduces the amount of data transferred over the network and improves performance. It also allows for more predictable and structured responses, making it easier to work with complex data structures.
+
+## Open Source Client Libraries
+
+To interact with the Centrifuge API, you can use a variety of open-source client libraries that simplify the process of querying GraphQL data. Here are some popular options:
+
+- **[Apollo Client](https://github.com/apollographql/apollo-client)**: A comprehensive state management library for JavaScript that enables you to manage both local and remote data with GraphQL. It is highly flexible and can be used with any JavaScript front-end framework, such as React, Angular, or Vue.js.
+- **[Relay](https://github.com/facebook/relay)**: A JavaScript framework for building data-driven React applications powered by GraphQL. Relay is known for its performance and scalability, making it suitable for large applications.
+- **[urql](https://github.com/urql-graphql/urql)**: A lightweight and flexible GraphQL client for React that is easy to set up and use. It provides a simple API for querying and mutating data.
+- **[graffle](https://github.com/graffle-js/graffle)**: Minimal GraphQL client supporting Node and browsers for scripts or simple apps.
+
+These libraries provide robust solutions for interacting with GraphQL APIs, each with its own strengths and use cases. Choose the one that best fits your project's requirements and your team's familiarity with the technology stack.
```

Comment:
For the target audience, I don't think a description of GraphQL client libraries is necessary.
```suggestion
```